release_notes/ocp-4-18-release-notes.adoc

```diff
@@ -3360,7 +3360,7 @@ $ oc adm release info 4.18.14 --pullspecs
 
 * Previously, a missing `afterburn` package resulted in the failure of the `gcp-hostname.service`, which caused the `scale-up` job to fail, impacting end-user deployments. With this release, the `afterburn` package is installed in the {op-system-base} `scale-up` job. This fix enables a successful `scale-up` action, resolving the `gcp-hostname` service failure. (link:https://issues.redhat.com/browse/OCPBUGS-55158[OCPBUGS-55158])
 
-* Previously, there was no communication between a `localnet` pod and a pod in the default network when both pods were on the same node. With this release, an update fixes the communication problem when pods are on the same node. (link:https://issues.redhat.com/browse/OCPBUGS-55016[OCPBUGS-55016])
+* Previously, a pod with a secondary interface in an OVN-Kubernetes `Localnet` network that was plugged into a `br-ex` interface bridge was out of reach by other pods on the same node, but used the default network for communication. The communication between pods on different nodes was not impacted. With this release, the communication between a `Localnet` pod and a default network pod running on the same node is possible, however the IP addresses that are used in the `Localnet` network must be within the same subnet as the host network. (link:https://issues.redhat.com/browse/OCPBUGS-55016[OCPBUGS-55016])
```

Comment:
```suggestion
* Previously, a pod with a secondary interface in an OVN-Kubernetes `Localnet` network that was plugged into a `br-ex` interface bridge was out of reach by other pods on the same node. The pod used the default network for communication. The communication between pods on different nodes was not impacted. With this release, the communication between a `Localnet` pod and a default network pod running on the same node is possible. However the IP addresses that are used in the `Localnet` network must be within the same subnet as the host network. (link:https://issues.redhat.com/browse/OCPBUGS-55016[OCPBUGS-55016])
```

Minor change to break the longer sentences.

Reply:
Uhm no, this changes the meaning of the original sentence. How about something along the lines of:

> Previously, a pod with a secondary interface in an OVN-Kubernetes Localnet network (mapped to the br-ex bridge) could not communicate with pods on the same node that used the default network for connectivity. However, communication between pods on different nodes remained unaffected.
> 
> With this release, Localnet pods can now communicate with default-network pods on the same node, provided that the Localnet IP addresses are within the same subnet as the host network.